 add_stylesheet was renamed to add_css_file from Sphinx 1.8
Index: kea-1.9.5/doc/sphinx/conf.py
===================================================================
--- kea-1.9.5.orig/doc/sphinx/conf.py
+++ kea-1.9.5/doc/sphinx/conf.py
@@ -228,6 +228,6 @@ def run_generate_docs(_):
 
 # custom setup hook
 def setup(app):
-    app.add_stylesheet('kea.css')
+    app.add_css_file('kea.css')
 
     app.connect('builder-inited', run_generate_docs)
